Documentation/badges/badge.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="/components/badge.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Badge Dexterous UI</title>
+  </head>
+  <body>
+    <h2>Badge</h2>
+    <p>Badge is used to show some additional information to the user.</p>
+
+    <h3>Badge on icon</h3>
+    <div class="badge-container">
+      <div class="icon-container">
+        <div class="badge align-badge">4</div>
+        <button class="btn btn-icon">
+          <i class="far fa-shopping-cart"></i>
+        </button>
+      </div>
+
+      <div class="icon-container">
+        <div class="badge align-badge">4</div>
+        <button class="btn btn-icon"><i class="fad fa-bell"></i></button>
+      </div>
+
+      <div class="icon-container">
+        <div class="badge align-badge">4</div>
+        <button class="btn btn-icon"><i class="fal fa-download"></i></button>
+      </div>
+    </div>
+
+    <h3>Badges on Avatar</h3>
+    <div class="avatar-container background">
+      <div>
+        <div class="avatar avatar-s">
+          <img src="/assets/minion-avatar.jpg" alt="" />
+        </div>
+        <div class="badge badge-s">
+        </div>
+      </div>
+
+      <div>
+        <div class="avatar avatar-m">
+          <img src="/assets/minion-avatar.jpg" alt="" />
+        </div>
+        <div class="badge badge-m badge-offline">
+        </div>
+      </div>
+
+      <div>
+        <div class="avatar avatar-l">
```

Comment:
small suggestion class name avatar-lg is easy to understand from avatar-l

Reply:
made suggested changes.